EIPS/eip-1155.md

```diff
@@ -39,7 +39,7 @@ pragma solidity ^0.5.9;
 
 /**
     @title ERC-1155 Multi Token Standard
-    @dev See https://eips.ethereum.org/EIPS/eip-1155
+    @dev See ./eip-1155.md
```

Comment:
I'm not so sure about this change. I assume this interface is copy pasted somewhere else and then this link becomes useless. Moreso, this is a Natspec definition which will be part of the contract metadata and this link will be useless.